src/txn/txn_rollback_to_stable.c

```diff
@@ -298,27 +298,23 @@ __rollback_check_if_txnid_non_committed(WT_SESSION_IMPL *session, uint64_t txnid
 
     /*
```

Comment:
@kommiharibabu Not sure why we use `__rollback_check_if_txnid_non_committed `? Can we change it `__rollback_check_id_visible`?

Reply:
Or should we extract a separate function to share the code with `__txn_id_visible` to prevent this divergence from happening again?

Reply:
Yes, it can be changed. You need to change all the callers to do the opposite.